### PR TITLE
MAT-4396-revision: Revise measuregroups page for compliance

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-4396](https://jira.cms.gov/browse/MAT-4396)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [x] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [x] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Review by sb-prateekkeerthi — DISMISSED** [review]

File: src/components/measureGroups/MultipleSelectDropDown.tsx [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import React from "react";
+import React, { ReactNode } from "react";
 import tw from "twin.macro";
 import OutlinedInput from "@mui/material/OutlinedInput";
 import MenuItem from "@mui/material/MenuItem";
 import FormControl from "@mui/material/FormControl";
 import Select from "@mui/material/Select";
 import { Checkbox, Link, ListItemText } from "@mui/material";
 import { InputLabel } from "@madie/madie-design-system/dist/react/";
+import { boolean } from "yup/lib/locale";
```
Comment (adongare) [inline src/components/measureGroups/MultipleSelectDropDown.tsx]: is this different from typescript boolean?

File: src/components/measureGroups/MultipleSelectDropDown.tsx [hunk 2]
```diff
@@ -1,11 +1,12 @@
-import React from "react";
+import React, { ReactNode } from "react";
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MultipleSelectDropDown.tsx]: Also we do have unused imports

File: src/components/measureGroups/MeasureGroups.tsx [hunk 3]
```diff
@@ -14,8 +14,10 @@ import {
   Link,
   Typography,
   Divider,
+  FormHelperText,
   Tabs,
   Tab,
+  TextField,
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx:20]: I believed we are using test fields from design systems !!

Comment (RohitKandimalla) [reply src/components/measureGroups/MeasureGroups.tsx:20]: ooh I don't think there is any usage of this component. nvm

File: src/components/measureGroups/MultipleSelectDropDown.tsx (hunk 1 above)
Comment (mcmcphillips) [reply src/components/measureGroups/MultipleSelectDropDown.tsx]: It looks like some built in utility. vscode auto added it at some point by accident

**Review by jkotanchik-SB — APPROVED** [review]

**Pull request merged**